impl/jamtemplate/common/game_state.cpp

```diff
@@ -50,8 +50,6 @@ void GameState::internalCreate()
 
 void GameState::internalUpdate(float elapsed)
 {
-    getGame()->gfx().window().updateGui(elapsed);
```

Comment:
unrelated to fixed update. Gui considerations should never have been part of `GameState` anyway.